.eslintrc.json

```diff
@@ -88,6 +88,14 @@
     ],
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/require-await": "off",
+    "@typescript-eslint/consistent-type-imports": [
```

Comment:
This is the new rule replacing the deprecated `importsNotUsedAsValues` typescript config.